### ISSUE TITLE
Allow usage of Consent.sourceAttachment

### ISSUE THREAD

**Issue by unixoid** [ehealthsuisse/ch-epr-fhir#243]
For using FHIR as a technical basis of a Policy Repository, it may be advantageous to store original XACML policy sets directly in FHIR Consent resources -- for that, the attribute Consent.sourceAttachment shall be enabled. 

A solution proposal is submitted in the pull request #241.

**Comment by oliveregger** [comment]
QL/MS/OE: need to discuss that in a followup meeting with Dmytro as the interface should not have the Consent and sourceAttachment in XACML which would be a duplication. Martin needs to have further discussion on the usecase.

**Comment by oliveregger** [comment]
@unixoid could you join the call on jan 30 2pm and elaborate on the usecase why it may be advantageous to store original XACML policy sets?

**Comment by unixoid** [comment]
@oliveregger Yes I can, please send me the invitation (or did you mean the FHIR.CH call on Jan 29?).

**Comment by unixoid** [comment]
As our project does not require this feature anymore, I withdraw the feature request.
